crates/store/src/server/api.rs

```diff
@@ -411,6 +411,11 @@ impl api_server::Api for StoreApi {
             .await
             .map_err(internal_error)?;
 
+        let (block_header, _) =
+            self.state.get_block_header(None, false).await.map_err(internal_error)?;
```

Comment:
This works, but I think we'll do an extra database access here which we may be able to avoid. Specifically, we could add something like `State::get_chain_length()` which would return the current length of the chain. Internally, this can be implemented by getting the number of nodes in the MMR stored in the `InnerState`.

Reply:
Turns out it was already implemented under the name of `latest_block_num()` in the State using the method that you described there. I will use that.

Reply:
Ah - nice! I forgot about this one.